views/editQuiz.handlebars

```diff
@@ -0,0 +1,36 @@
+<body>
+    {{> header}}
+
+    <div class="edit_quiz">
+        <div class="edit_intro">
+            <input placeholder="Title..."></input>
+            <textarea placeholder="Description..."></textarea>
+            <div class="edit_quiz_tags">
+                <p>Tags (separated by space)</p>
+                <input placeholder="Tags..."></input>
+            </div>
+        </div>
+        <div class="edit_question">
+            <p>Edit Questions</p>
+        </div>
+        <div class="edit_questions_container">
+            <div class="edit_flash_card">
+                <button type="button" class="remove_card_button" class="clickable">x</button>
```

Comment:
Please put both classes in the same string. You use a space to separate multiple classes. ` class="remove_card_button clickable"`